docs/TranslatableInputs.md

```diff
@@ -170,6 +172,58 @@ For instance, you can use `direction: 'row'` to display the inputs side by side
 
 ![TranslatableInputs with direction row](./img/TranslatableInputs-row.png)
 
+## `readOnly`
+
+If `true`, the input is in read-only mode and the user can't change the value.
+
+{% raw %}
+
+```jsx
+<TranslatableInputs
+    locales={['en', 'fr']}
+    readOnly
+>
+    <TextInput source="title" />
+    <TextInput source="description" />
+</TranslatableInputs>
+```
+
+{% endraw %}
+
+## `disabled`
+
+If `true`, the input is disabled and the user can't change the value.
+
+{% raw %}
+
+```jsx
+<TranslatableInputs
+    locales={['en', 'fr']}
+    disabled
+>
+    <TextInput source="title" />
+    <TextInput source="description" />
+</TranslatableInputs>
+```
+
+{% endraw %}
+
+**Tip**: The form framework used by react-admin, react-hook-form, [considers](https://github.com/react-hook-form/react-hook-form/pull/10805) that a `disabled` input shouldn't submit any value. So react-hook-form sets the value of all `disabled` inputs to `undefined`. As a consequence, a form with a `disabled` input is always considered `dirty` (i.e. react-hook-form considers that the form values and the initial record values are different), and it triggers [the `warnWhenUnsavedChanges` feature](./EditTutorial.md#warning-about-unsaved-changes) when leaving the form, even though the user changed nothing. The workaround is to set the `disabled` prop on the underlying input component, as follows:
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought (non-blocking):** ‏same question as before :thinking: :thinking: